CMakeLists.txt

```diff
@@ -51,11 +51,14 @@ list(APPEND LIBS ${DEATHKNELL})
 
 find_library(TCMALLOC tcmalloc PATHS /usr/local/probe/lib )
 # Appended first instead
+find_library(NANO_MSG nanomsg PATHS /usr/local/lib )
+list(APPEND LIBS ${NANO_MSG})
 
-
+find_library(MPI mpi PATHS /usr/lib64/openmpi/lib )
+find_library(MPI_CXX mpi_cxx PATHS /usr/lib64/openmpi/lib )
 
 MESSAGE(" PROJECT_SRC = : ${PROJECT_SRC}")
-include_directories(${PROJECT_SRC})
+include_directories(${PROJECT_SRC} /usr/include/openmpi-x86_64/)
```

Comment:
Does this need openMPI to be added to baseInstall or Thirdparty?  

We might want to consider installing it as an RPM in the netmon common directory /us